noir_stdlib/src/collections/vec.nr

```diff
@@ -0,0 +1,14 @@
+
+// These methods are all stubs currently and aren't implemented internally yet.
+// For a similar reason, no constructor for Vec is exposed yet since the type
+// is still in-progress.
+impl<T> Vec<T> {
+    #[builtin(vec_get)]
```

Comment:
Can we document what these methods are doing? (Even though it is somewhat self-explanatory, I think folks will use our std-lib as the standard for documentation)

(Eventually we will need to document the rest of our stdlib, but its too much right now)